template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl

```diff
@@ -26,12 +26,21 @@ dbutils.widgets.dropdown("env", "dev", ["dev", "staging", "prod"], "Environment
 dbutils.widgets.text("input_table_name", "", label="Input Table Name")
 # Delta table to store the output predictions.
 dbutils.widgets.text("output_table_name", "", label="Output Table Name")
+{{- if (eq .input_include_models_in_unity_catalog "no") }}
```

Comment:
It will be great if DECO team fixed it. Could you double check?
Previously I have to use  
```
eq .input_include_models_in_unity_catalog `no`
```

When I did the migration, `"no"` did not work.

Reply:
it looks like they have indeed fixed it!

Reply:
it looks like they have indeed fixed it!